test/__mocks__/svgMock.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { SVGProps } from 'react';
+
+export const ReactComponent = React.forwardRef<
+  SVGSVGElement,
+  SVGProps<SVGSVGElement>
+>((props, ref) => (
+  <svg
+    ref={ref}
+    /* eslint-disable-next-line react/jsx-props-no-spreading */
+    {...props}
```

Comment:
I'm disabling the `jsx-props-no-spreading` rule here to inherit any props svgr passes to the component. I think that mapping them manually is quite an overkill.